WPFHospitalEditor/HospitalMap.xaml.cs

```diff
@@ -156,12 +164,26 @@ private void setMedicationNameComboBox()
             }
         }
 
+        private void setDoctorNameComboBox()
+        {
+            foreach (DoctorDto docDto in doctorServerController.GetDepartmentDoctors(regularExaminationDepartment))
+            {
+                doctorsComboBox.Items.Add(docDto.Name + " " + docDto.Surname);
+            }
+        }
+
         private Boolean IsPatientLogged()
         {
             if (role == Role.Patient) return true;
             return false;
         }
 
+        private Boolean IsSecretaryLogged()
+        {
+            if (role == Role.Secretary) return true;
+            return false;
+        }
+
```

Comment:
You can create one function for every role, like:
`IsRoleLogged(Role role2){
  return role==role2;
}`
You can name parameter of this pseudo function much better.

Reply:
fixed